core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -152,102 +144,18 @@ public void init() throws StoreInitializationException {
     }
   }
 
-  public static void addSecurityConfigsToClientProperties(SchemaRegistryConfig config,
-                                                          Properties props) {
-    addSslConfigsToClientProperties(config, props);
-    addSaslConfigsToClientProperties(config, props);
+  public static void addSchemaRegistryConfigsToClientProperties(SchemaRegistryConfig config,
+                                                                Properties props) {
+    props.putAll(config.originalProperties());
```

Comment:
yes but this would allow users to just configure the one with non prefix and use it for kafka store..Thought it would help reduce configs in an enterprise security scenario?